stock_quant_package_fast_move/models/stock_quant_package.py

```diff
@@ -27,15 +27,15 @@ def check_source_location(self):
                 _("All packages must have the same source location.")
             )
 
-    def _move_to_location(self, location, destination_package=None, no_validate=False):
+    def _move_to_location(self, location, destination_package=None, validate=False):
         """
         Move packages to a specified location.
 
         Parameters:
         - location (recordset of stock.location): The destination location.
         - destination_package (recordset of stock.quant.package, optional):
         Optional destination package. If provided, it must belong to the specified location.
-        - no_validate (boolean, optional):
+        - validate (boolean, optional):
```

Comment:
If set to True, the created picking will be validated. Otherwise it will remain in the "Ready" state.

Reply:
Thank you for your review. Could you please check it again?